src/mlpack/bindings/python/tests/test_python_binding_main.cpp

```diff
@@ -30,6 +30,7 @@ PARAM_DOUBLE_IN_REQ("double_in", "Input double, must be 4.0.", "d");
 PARAM_FLAG("flag1", "Input flag, must be specified.", "f");
 PARAM_FLAG("flag2", "Input flag, must not be specified.", "F");
 PARAM_MATRIX_IN("matrix_in", "Input matrix.", "m");
+PARAM_MATRIX_IN("smatrix_in", "Input matrix.", "");
```

Comment:
It seems to me like we could use the regular `matrix_in` parameter here.  Or is there some reason we need `smatrix_in` and `smatrix_out`?

Reply:
I have to test pandas.Series as matrix which is 1 dim so i use another parameter name `smatrix`

Reply:
Ah, okay, understood.